integration/tests/access/cohort4/access_test.go

```diff
@@ -106,51 +97,55 @@ func (s *AccessSuite) TestAPIsAvailable() {
 		defer cancel()
 
 		grpcAddress := s.net.ContainerByName(testnet.PrimaryAN).Addr(testnet.GRPCPort)
-		conn, err := grpc.DialContext(ctx, grpcAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		conn, err := grpc.NewClient(grpcAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
 		require.NoError(t, err, "failed to connect to access node")
 		defer conn.Close()
 
 		client := accessproto.NewAccessAPIClient(conn)
 
-		_, err = client.Ping(s.ctx, &accessproto.PingRequest{})
+		_, err = client.Ping(ctx, &accessproto.PingRequest{})
 		assert.NoError(t, err, "failed to ping access node")
 	})
 }
 
-// TestSignerIndicesDecoding tests that access node uses signer indices' decoder to correctly parse encoded data in blocks.
+// runTestSignerIndicesDecoding tests that access node uses signer indices' decoder to correctly parse encoded data in blocks.
 // This test receives blocks from consensus follower and then requests same blocks from access API and checks if returned data
 // matches.
-func (s *AccessSuite) TestSignerIndicesDecoding() {
-
+// CAUTION: must be run last if running multiple tests using the same network since it stops the containers.
+func (s *AccessSuite) runTestSignerIndicesDecoding() {
 	container := s.net.ContainerByName(testnet.PrimaryAN)
 
 	ctx, cancel := context.WithCancel(s.ctx)
 	defer cancel()
 
 	// create access API
 	grpcAddress := container.Addr(testnet.GRPCPort)
-	conn, err := grpc.DialContext(ctx, grpcAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	conn, err := grpc.NewClient(grpcAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	require.NoError(s.T(), err, "failed to connect to access node")
 	defer conn.Close()
 
 	client := accessproto.NewAccessAPIClient(conn)
 
-	// query latest finalized block
-	latestFinalizedBlock, err := MakeApiRequest(client.GetLatestBlockHeader, ctx, &accessproto.GetLatestBlockHeaderRequest{
-		IsSealed: false,
-	})
-	require.NoError(s.T(), err)
-
+	// query latest finalized block. wait until at least one block has been finalized.
+	// otherwise, we may get a block that is both the latest finalized and sealed block, which will break
+	// the checks below.
```

Comment:
this started failing because `GetLatestBlockHeader` returns the block status based on which `state` method was used to get it (`Finalized()` vs `Sealed()`). this works in all cases except for the root block which is both the latest finalized and sealed block until the first block is finalized.

This does point to a corner case where calling `GetLatestBlockHeader` with `isSealed=false` on an Access node that has not locally finalized its first block would report the returned block as finalized, even though it is sealed. Given this is a narrow corner case and the impact is thinking a sealed block is not yet sealed, I don't think it's worth adding a runtime check for all block lookups.

I'm also OK to add it if others feel it's important.